### PR TITLE
fix empty initialValues causing error

### LINKED ISSUE #71: Application crashes with empty initialValues object
**Describe the bug**
For some forms, I don't need any initialValues but then the app crashes.

**To Reproduce**

Works
 ```
  initialValues: {
      lastName: ''
    }
```

Doesn't work
 ```
  initialValues: {}
```

**Expected behavior**
It should not throw an error and crash the application.

**Stacktrace**
```
Uncaught TypeError: Cannot destructure property 'form' of 'createForm(...)' as it is undefined.
    at instance$e (Registration.svelte:9)
    at init (index.mjs:1471)
    at new Registration (Registration.svelte:185)
    at Array.create_if_block_1 (Route.svelte:31)
    at create_if_block (Route.svelte:30)
    at Object.update [as p] (Route.svelte:29)
    at update (index.mjs:765)
    at flush (index.mjs:733)
    at init (index.mjs:1501)
    at new App (App.svelte:22)
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.10
 - Browser: Brave
 - Version 1.17.73

### PR DESCRIPTION
**Pull request by subpx**
Fixes #71

Removed isInitialValuesValid check. Don't think it made much sense having it, especially since the default value for initialValues is an empty object which doesn't pass the check.

Needed to remove 1 test. Sorry

### CODE REVIEW

File: test/library.spec.js [hunk 1]
```diff
@@ -41,19 +41,6 @@ describe('createForm', () => {
     instance = getInstance();
   });
 
-  describe('config', () => {
```
Comment (larrybotha) [inline test/library.spec.js:44]: @subpx it'd be good to test here that `createForm` doesn't throw when `initialValues` is `undefined` for future safety, something like:

```javascript
   it('handles undefined initialValues', () => {
      const initialValues = undefined;
      const config = {initialValues, onSubmit: jest.fn()};

      expect(() => createForm(config)).not.toThrow();
    });
```

Comment (subpx) [reply test/library.spec.js:44]: OK I've added that test

**Comment by larrybotha** [comment]
Thanks @subpx - LGTM! Awaiting 1 more review before merge.

**Review by tiaanduplessis — APPROVED** [review]

**Comment by mhatvan** [comment]
Great work guys!

**Pull request merged**